packages/web-app-files/src/components/Search/List.vue

```diff
@@ -287,6 +325,33 @@ export default defineComponent({
       })) || []
     )
 
+    const mediaTypeMapping = {
+      file: { label: $gettext('File'), icon: 'txt' },
+      folder: { label: $gettext('Folder'), icon: 'folder' },
+      document: { label: $gettext('Document'), icon: 'doc' },
+      spreadsheet: { label: $gettext('Spreadsheet'), icon: 'xls' },
+      presentation: { label: $gettext('Presentation'), icon: 'ppt' },
+      pdf: { label: $gettext('PDF'), icon: 'pdf' },
+      image: { label: $gettext('Image'), icon: 'jpg' },
+      video: { label: $gettext('Video'), icon: 'mp4' },
+      audio: { label: $gettext('Audio'), icon: 'mp3' },
+      archive: { label: $gettext('Archive'), icon: 'zip' }
+    }
+    const availableMediaTypeValues: FileCategoryKeyword[] = []
+    unref(mediaTypeCapability).keywords?.forEach((key: string) => {
+      if (!mediaTypeMapping[key]) {
+        return
+      }
+      availableMediaTypeValues.push({
+        id: key,
+        ...mediaTypeMapping[key]
+      })
+    })
```

Comment:
Could you please apply an order? I'd suggest:
- Folder
- File
- all other options sorted lexicographically

I'd put `Folder` and `File` to the top because they're somewhat special.

Reply:
Sorry, my bad. The order is specified in the issue and should come from the backend. You did that correctly. 👍

Reply:
alright 👍🏼